aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js

```diff
@@ -0,0 +1,321 @@
+  },
+  {
+    name: 'Beacon A',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon B',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon C',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon D',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+];
+
+const getPageDetails = pageName => pages.find(page => page.name === pageName);
+
+export const HubSpokeTilesExample = ({ mobile }) => {
+  const [currentPage, setCurrentPage] = React.useState(getPageDetails('Home'));
+  const [subPages, setSubPages] = React.useState();
+
+  React.useEffect(() => {
+    const pageDetails =
+      currentPage.children &&
+      currentPage.children.map(child =>
+        pages.find(page => page.name === child),
+      );
+    setSubPages(pageDetails);
+  }, [currentPage]);
+
+  const { children, layout, name } = currentPage;
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
+      <ResponsiveContext.Consumer>
+        {size => (
+          <AppContainer>
+            <AppHeader
+              currentPage={currentPage}
+              setCurrentPage={setCurrentPage}
+              size={size}
+            />
+            <Box
+              pad={{
+                horizontal: size !== 'small' ? 'large' : 'medium',
+                vertical: 'small',
+              }}
+            >
+              <Heading level={2} margin={{ top: 'none', bottom: 'medium' }}>
+                {name}
+              </Heading>
+              {layout === 'tiles' && (
+                <GridLayout
+                  items={subPages}
+                  onClick={page => setCurrentPage(getPageDetails(page))}
+                />
+              )}
+              {layout === 'list' && <ListLayout items={children} />}
+              {layout === 'panes' && <PanesLayout items={children} />}
+            </Box>
+          </AppContainer>
+        )}
+      </ResponsiveContext.Consumer>
+    </ResponsiveContext.Provider>
+  );
+};
+
+HubSpokeTilesExample.propTypes = {
+  mobile: PropTypes.bool,
+};
+
+const AppContainer = ({ ...rest }) => {
+  const size = React.useContext(ResponsiveContext);
+
+  return (
+    <Box
+      background="background-back"
+      fill="vertical"
+      height={size === 'small' ? { max: 'large' } : undefined}
+      width={size === 'small' ? 'medium' : '100%'}
+      overflow="auto"
+      {...rest}
+    />
+  );
+};
+
+const AppHeader = ({ currentPage, setCurrentPage }) => {
+  const size = React.useContext(ResponsiveContext);
+
+  return (
+    <Header
+      fill="horizontal"
+      height="xsmall"
+      pad={{ horizontal: size !== 'small' ? 'large' : 'medium' }}
+    >
+      <AppIdentity
+        name={`Service ${
+          typeof currentPage.parent !== 'undefined'
+            ? `| ${currentPage.parent}`
```

Comment:
This breadcrumb type layout was a little unexpected to me -- I hadn't noticed the title changed so it took me a while to figure out how to go back up one-level. I guess I was expecting more of a "back to [parent page name]" button just above the page content.

Screen Shot 2020-05-06 at 9.25.40 AM

Reply:
Agree. I was trying to apply this pattern that was submitted in the #hpexs-designreviewers channel, but I too feel like it gets lost. https://hpe.slack.com/archives/CTUQ1PQDS/p1585679610002700

https://designer.grommet.io/dashboard?id=GLC-Page-Naming-and-Breadcrumbs-edgar-circenis-hpe-com

I like what we are doing on the site better:
![Screen Shot 2020-05-06 at 10 35 40 AM](https://user-images.githubusercontent.com/1756948/81203581-5a804700-8f85-11ea-86d7-4b367874c799.png)

Reply:
Will adjust.